scripts/notifyRewardAmount2.js

```diff
@@ -0,0 +1,78 @@
+// ES5 style
+const find = require('lodash/find');
+const readlineSync = require('readline-sync');
+
+const Multirewards = require('../build/contracts/MultiRewards.json');
+const IERC20 = require('../build/contracts/IERC20.json');
+
+const { getConnex } = require('./utils');
+
+const notifyRewardAmount = async ({
+  connex,
+  multiRewards,
+  network,
+  rewardAmount,
+  rewardToken,
+}) => {
+  const approveABI = find(IERC20, { name: 'approve' });
+  const approveMethod = connex.thor.account(rewardToken).method(approveABI);
+
+  const notifyRewardAmountABI = find(Multirewards.abi, { name: 'notifyRewardAmount' });
+  const notifyRewardAmountMethod = connex.thor.account(multiRewards).method(notifyRewardAmountABI);
+
+  const approveClause = approveMethod.asClause(multiRewards, rewardAmount);
+  const notifyRewardAmountClause = notifyRewardAmountMethod.asClause(rewardToken, rewardAmount);
```

Comment:
we need to be super careful of the rewardAmount here. When we pass in the argument from the `refreshAllPools` script, I think it does not contain 18 decimals. So we need to convert into the correct number of decimals before calling the methods. If not we'll be off by 18 decimal places